stable/fluent-bit/values.yaml

```diff
@@ -42,6 +42,10 @@ trackOffsets: false
 ## Ref: https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/#priorityclass
 priorityClassName: ""
 
+## Run the container as a privileged user. Necessary for certain input plugins.
+## daemonSet.podSpec.container.securityContext.privileged
+runAsPrivileged: false
```

Comment:
would it be more flexible to just have a `securityContext` map, which can be toYaml'ed into the descriptor? That way you can control all aspects of the sec. context.

Reply:
Yes, that's not a bad idea. I can do that. It just relies on users being more k8 spec friendly.